### PR TITLE
Prepare for moving building to github actions

### PR DESCRIPTION
**Pull request by sebastianscatularo**
- Added production.yml with steps to build and push an artifact to the github npm registry
- Add minor change at `src/components/Footer.tsx` to display the version
- Add failure at deploy time, due that the deploy hook is not yet implemented

### CODE REVIEW

File: public/package.json
Comment (valentinoConti) [inline public/package.json:1]: Can I ask just out of curiosity what does this file do ?

**Review by valentinoConti — APPROVED** [review]

File: public/package.json
Comment (sebastianscatularo) [reply public/package.json:1]: it is necessary to publish the build folder as a package with out all the dependencies, this is a workaround to speed up Cloudflare build, we do not need to install dependencies, as we are shipping a pre-build package

Comment (valentinoConti) [reply public/package.json:1]: excellent!

Comment (sebastianscatularo) [reply public/package.json:1]: it is removed at deployment time https://github.com/XLabs/portal-bridge-ui/blob/release/package.json#L6

**Comment by fchantada** [comment]
Force Merge due to the current problem with Cloudflare page build.

**Pull request merged**